include/fc/io/cfile.hpp

```diff
@@ -49,6 +51,9 @@ class cfile {
    ///         "ab+" - open for binary update - create if does not exist
    ///         "rb+" - open for binary update - file must exist
    void open( const char* mode ) {
+       if(_open)
```

Comment:
For correctness, I would suggest setting _open back to false.

Reply:
good call.